addon/services/current-user.js

```diff
@@ -63,4 +63,16 @@ export default Ember.Service.extend({
             promise: this.load().catch(() => null),
         });
     }),
+
+    /**
+     * @property esPreferenceKey
+     * @return {String}
+     */
+    esPreferenceKey: Ember.computed('currentUserId', function() {
```

Comment:
A comment might be good here explaining why we need this.

Reply:
Just a drive-by thought, for what it's worth:

It's generally nicer to name something after what it *is* or what it *does*, not how/when it's used. This is a user- or session-specific key that is currently used only for ES preference, but could be used for other things in the future.

Something like `userOrSessionKey` (or something better, I don't know, naming is hard) would be less likely to cause confusion/annoyance/work in the future. Then, as @laurenbarker said, a comment would be a good place to explain how we expect to use it, why it exists.